drivers/clock_control/renesas/r8a77951-cpg-mssr.c

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
The convention is to use `_` as separator in file names - not `-`.

Reply:
I used these file name to match Linux file names, are we not trying to be as close as possible from Linux code base if possible ?
I'll do the change then, thanks for your message.

Reply:
renamed all files !